README.md

```diff
@@ -26,7 +26,13 @@ export PG_PASSWORD="example_password"
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
-Ready to run in production? 
+## Ready to run in production?
 * Please [check the phoenix deployment guides](http://www.phoenixframework.org/docs/deployment).
-* Set the environment variable (SECRET_KEY_BASE) used for session cookies.
-* Set the environment variable (GUARDIAN_JWK_KEY) for the secret key which guardian uses for JWT tokens.
+
+### Environment Variables to set
+
+* SECRET_KEY_BASE - Used to generate session cookies.
+* GUARDIAN_JWK_KEY - Used to generate JWT tokens.
+* PAIRMOTRON_EMAIL_DOMAIN - The domain configured in Mailgun to send password reset email.
```

Comment:
they are not required, since `System.get_env/1` returns nil if the config isn't found. Which isn't actually used by Bamboo until it tries to send an email (at which point it obviously fails). So it really only fails if you try to request a password_reset and it isn't properly configured.